static/elements/chromedash-approvals-dialog.js

```diff
@@ -230,6 +255,63 @@ class ChromedashApprovalsDialog extends LitElement {
     }
   }
 
+  renderConfigWidgets(approvalDef) {
+    const fieldId = approvalDef.id;
+    const isOpen = this.showConfigs.has(fieldId);
+    if (!isOpen) {
+      return nothing;
+    }
+
+    const config = this.configs.find(c => c.field_id == fieldId);
+    const owners = (config && config.owners || []).join(', ');
```

Comment:
This line is not very intuitive. I understand it used as non-Boolean operands but according to the API `[a5 = 'Cat' && 'Dog'      // t && t returns "Dog"]`. It is not clear what precedence is here.

Reply:
I rewrote these as conditional expressions (like `c ? x : y`) and I think that that is an improvement.  Thanks.